ons_alpha/bulletins/models.py

```diff
@@ -111,9 +111,13 @@ class BulletinPage(BundledPageMixin, RoutablePageMixin, BasePage):
         index.AutocompleteField("get_admin_display_title"),
     ]
 
+    @property
```

Comment:
Query: would making this a @cached_property be more performant?

Reply:
It would be for computational-intensive methods, but this is quite simple hence negligible or worse here as `get_parent` also stores a cache itself.

Reply:
Noted, thanks.